lib/src/nakama_client/nakama_client.dart

```diff
@@ -103,4 +103,11 @@ abstract class NakamaBaseClient {
     bool? forward,
     String? cursor,
   });
+
+  Future<Map> postRPC({
```

Comment:
While adopting the original SDKs the naming should be like `client.rpc` instead of `postRpc`: https://heroiclabs.com/docs/nakama/client-libraries/unity/#remote-procedure-calls